{"uuid": "6966df47-60bf-4406-ad23-181f0720b137", "children": ["6cf213c8-667f-456f-8280-d53b58a6dd28", "5f7169ac-4da0-421c-a08c-52a48f8a47ed"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1596044155813, "stop": 1596044155813}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'TestCase' has no attribute 'driver'\n", "trace": "  File \"D:\\python\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\python.py\", line 705, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\appium_PO_kaoyan\\test_case\\test_cases.py\", line 23, in teardown_class\n    Common(self.driver).send_email('../report/pytest_report.html')\n"}, "start": 1596044187766, "stop": 1596044187768}], "start": 1596044155813, "stop": 1596044187768}